pages/linux/cmus.md

```diff
@@ -0,0 +1,24 @@
+# cmus
+
+> Commandline C* Music Player.
+> Arrow keys to navogate, `<enter/return>` to select, numbers 1-8 switch between different views.
```

Comment:
typo: "navogate". Also, please start the sentence with "Use...", and add "and" before "numbers".